go/v1beta1/storage/types.go

```diff
@@ -31,79 +31,84 @@ type esSearchResponseHit struct {
 
 // Elasticsearch /_search query
 
-type esSearch struct {
-	Query *filtering.Query       `json:"query,omitempty"`
-	Sort  map[string]esSortOrder `json:"sort,omitempty"`
+type EsCollapse struct {
+	Field string `json:"field,omitempty"`
 }
 
-type esSortOrder string
+type EsSearch struct {
+	Query    *filtering.Query       `json:"query,omitempty"`
+	Sort     map[string]EsSortOrder `json:"sort,omitempty"`
+	Collapse *EsCollapse            `json:"collapse,omitempty"`
```

Comment:
This field and the `EsCollapse` struct are the only new additions, everything else is just a rename. 

We're using the [collapse](https://www.elastic.co/guide/en/elasticsearch/reference/current/collapse-search-results.html#collapse-search-results) field to dedup results.